modules/creating-infra-machines.adoc

```diff
@@ -31,6 +32,11 @@ spec:
     matchLabels:
       node-role.kubernetes.io/infra: ""
 ----
++
+[NOTE]
+====
+Custom machine config pools inherit machine configs from the worker pool. Custom pools use any machine config targeted for the worker pool, but add the ability to also deploy changes only targeted at the custom pool. Since a custom pool inherits resources from the worker pool, any change to the worker pool also affects the custom pool.
```

Comment:
I think this is a very good note. Should we perhaps put this somewhere else as well for general machineconfigpool management?

Reply:
Perhaps in a follow-up PR?